contracts/limitOrder/LimitOrderBook.sol

```diff
@@ -158,7 +179,27 @@ contract LimitOrderBook is
 
         _ordersStatus[orderHash] = ILimitOrderBook.OrderStatus.Cancelled;
 
-        emit LimitOrderCancelled(order.trader, order.baseToken, orderHash);
+        int256 positionSize;
```

Comment:
question, we need this piece of code bcs we need `positionSize` and `positionNotional` for the event, right? If so, all the info should be able to be found in our backend db, right? Why don't we just fetch these values from DB?

Reply:
Yes, limit order backend currently will delete cancelled limit orders though. But I think it's still good to know in subgraph.

Can ask Tempo not to delete cancelled orders, and frontend filters out cancelled ones.